modules/CMakeLists.txt

```diff
@@ -5,8 +5,13 @@ sofa_add_module(SofaSparseSolver SofaSparseSolver ON)      #
 sofa_add_module(SofaPreconditioner SofaPreconditioner ON)  # depends on SofaSparseSolver
 sofa_add_module(SofaHaptics SofaHaptics ON)
 
+find_package(Newmat QUIET)
+if(Newmat_FOUND)
+	sofa_add_module(SofaDenseSolver SofaDenseSolver ON)
+endif()
+
 if(SOFA_NO_OPENGL)
-    message("SOFA_NO_OPENGL flag prevents from using the SofaCUDA, SofaSimpleGUI and VolumetricRendering plugins")
+    message("SOFA_NO_OPENGL flag prevents from using the SofaOpenglVisual plugin")
```

Comment:
Not sure.

Reply:
For SofaCUDA, sure ;) I did a PR to remove the need of OpenGL
By the way, for me we should not show the message of support/not support at this place. It should be where SofaSimpleGUI is added